### PR TITLE
MES-4254: Copy Test Report

### PR DESCRIPTION
**Pull request by szabikr**
## Description

- Copying test report to Cat C
- Extracting reverse diagram analytics to common
- Adding the Cat C Screen names

## Checklist

- [x] PR title includes the JIRA ticket number
- [ ] Branch is rebased against the latest develop
- [ ] Code has been tested manually
- [ ] PR link added to JIRA ticket
- [ ] One review from each scrum team
- [ ] Squashed commit contains the JIRA ticket number

### CODE REVIEW

**Review by ammarhaiderbjss — APPROVED** [review]

File: src/pages/page-names.constants.ts [hunk 1]
```diff
@@ -45,3 +45,22 @@ export const CAT_BE = {
...
+  HEALTH_DECLARATION_PAGE: 'HealthDeclarationCatCPage',
+  OFFICE_PAGE: 'OfficeCatCPage',
+  PASS_FINALISATION_PAGE: 'PassFinalisationCatCPage',
+  REVERSE_DIAGRAM_PAGE: 'ReverseDiagramCatCPage',
+  TEST_REPORT_PAGE: 'TestReportCatCPage',
+  WAITING_ROOM_PAGE: 'WaitingRoomCatCPage',
+  WAITING_ROOM_TO_CAR_PAGE: 'WaitingRoomToCarCatCPage',
+  POST_DEBRIEF_HOLDING_PAGE: 'PostDebriefHoldingCatCPage',
+  REKEY_REASON_PAGE: 'RekeyReasonCatCPage',
+  REKEY_UPLOAD_OUTCOME_PAGE: 'RekeyUploadOutcomePage',
```
Comment (rossfauldsbjss) [inline src/pages/page-names.constants.ts:62]: Is there no cat c version of the rekey upload outcome page?

File: src/pages/test-report/cat-c/components/reverse-diagram-modal.cat-c/__tests__/reverse-diagram-modal.spec.ts [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+        store$.dispatch(new StartTest(123, TestCategory.BE));
+      });
+  }));
+
+  describe('DOM', () => {
+
+  });
+
+  describe('Class', () => {
+    const vehicleDetails: CatBEUniqueTypes.VehicleDetails = {
```
Comment (rossfauldsbjss) [inline src/pages/test-report/cat-c/components/reverse-diagram-modal.cat-c/__tests__/reverse-diagram-modal.spec.ts]: TODO to replace with CatCUniqueTypes

**Review by rossfauldsbjss — APPROVED** [review]
looks good

File: src/pages/page-names.constants.ts (hunk 1 above)
Comment (szabikr) [reply src/pages/page-names.constants.ts:62]: That should not even be in any category, as it's not a category specific page. let's solve that later

File: src/pages/test-report/cat-c/components/reverse-diagram-modal.cat-c/__tests__/reverse-diagram-modal.spec.ts (hunk 2 above)
Comment (szabikr) [reply src/pages/test-report/cat-c/components/reverse-diagram-modal.cat-c/__tests__/reverse-diagram-modal.spec.ts]: adding TODO now

**Pull request merged**